frontend/src/app/homepage-offer/homepage-offer.component.html

```diff
@@ -1,10 +1,10 @@
 <article class="offer col-xs-12">
   <div *ngIf="offer.image != null" class="offer-cover-image">
-    <a [routerLink]="['offers', offer.slug, offer.id]" class="offer-img"
+    <a [routerLink]="['offers', getShortenedOfferSlug(offer.slug), offer.id]" class="offer-img"
```

Comment:
Can You reason, why we should shorten slugs? Even if so, we should do that on backend, as that not a good idea to split slug logic, part done in Django and part of it done here.

Reply:
We can talk about this also. For now I will stash my changes of that for the future if needed.